doclet.c: In function `gtkdoc_director_scangobj':
doclet.c:819: error: expected `)' before `->' token
doclet.c:851: warning: assignment from incompatible pointer type
doclet.c:871: warning: assignment from incompatible pointer type

--- origsrc/valadoc/src/doclets/gtkdoc/doclet.vala	2010-05-12 21:49:43.000000000 -0500
+++ src/valadoc/src/doclets/gtkdoc/doclet.vala	2010-05-12 22:18:18.679478000 -0500
@@ -226,21 +226,21 @@ public class Gtkdoc.Director : Valadoc.D
 		pc_libs += "--libs";
 	
 		try {
-			string stderr;
+			string std_err;
 			int status;
 
 			string cflags;
-			Process.spawn_sync (null, pc_cflags, null, SpawnFlags.SEARCH_PATH, null, out cflags, out stderr, out status);
+			Process.spawn_sync (null, pc_cflags, null, SpawnFlags.SEARCH_PATH, null, out cflags, out std_err, out status);
 			if (status != 0) {
-				warning ("GtkDoc: pkg-config cflags error: %s\n", stderr);
+				warning ("GtkDoc: pkg-config cflags error: %s\n", std_err);
 				return false;
 			}
 			cflags = cflags.strip ();
 
 			string libs;
-			Process.spawn_sync (null, pc_libs, null, SpawnFlags.SEARCH_PATH, null, out libs, out stderr, out status);
+			Process.spawn_sync (null, pc_libs, null, SpawnFlags.SEARCH_PATH, null, out libs, out std_err, out status);
 			if (status != 0) {
-				warning ("GtkDoc: pkg-config libs error: %s\n", stderr);
+				warning ("GtkDoc: pkg-config libs error: %s\n", std_err);
 				return false;
 			}
 
